src/main/java/codeu/controller/keywords.java

```diff
@@ -0,0 +1,34 @@
+package codeu.controller;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+public class keywords {
+  //method breaks string into string array and then filters out stop words.
+  public static ArrayList<String> str2kw(String bio) {
+      String[] arr = bio.split("[^a-zA-Z]+");
```

Comment:
Why not split using space? Something like bio.split("\\s+")?

Reply:
I think this is a better regex filter, because it splits based on whether the character isn't a letter anymore. The downside is it will break up hyphenated words, but it won't attach other punctuation like "\s+" will i think.